dist/styles/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as button } from './buttons.css';
+export { default as form } from './form.css';
+export { default as link } from './links.css';
+export { default as util } from './utils.css';
```

Comment:
Need an empty line

Reply:
this is generated file by gulp, so I have no control.

Reply:
will take a look at a solution with gulp later.